misc/windows-deploy/Install-ECSAgent.ps1

```diff
@@ -0,0 +1,139 @@
+﻿# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the
+# License is located at
+#
+#	http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+[cmdletbinding()]
+Param (
+    [Parameter(Mandatory=$false)]
+    [ValidateNotNullOrEmpty()]
+    [string]$Path = "$($PSScriptRoot)",

```

Comment:
Is this because we want to override the installation path?  Is the intent that we put this script inside `%ProgramFiles%\Amazon\ECS` along with the `amazon-ecs-agent.exe`?

Reply:
Thats correct. Since the 'amazon-ecs-agent.exe' will be in the same extracted folder...